example/AndroidManifest.xml

```diff
@@ -4,19 +4,22 @@
           android:versionCode="1"
           android:versionName="1.0">
 
-    <uses-sdk android:minSdkVersion="7" android:targetSdkVersion="25"/>
```

Comment:
Note -- there's no point having the minSdkVersion and targetSdkVersion in the manifest when you can just put it in the gradle file.